exchanges/bitstamp/bitstamp_types.go

```diff
@@ -180,6 +180,29 @@ type CryptoWithdrawalResponse struct {
 	ID int64 `json:"withdrawal_id"`
 }
 
+// OpenBankWithdrawalRequest holds the request information for sending a bank withdrawal request
+type OpenBankWithdrawalRequest struct {
```

Comment:
Suggestion: seeing as it's not used in the request body 
```suggestion
type OpenBankWithdrawalParams struct {
```